bin/echo.c

```diff
@@ -250,6 +250,9 @@ int print_connection_info(struct s2n_connection *conn)
         printf("\n");
     }
 
+    printf("Wire Bytes In: %lu\n", s2n_connection_get_wire_bytes_in(conn));
+    printf("Wire Bytes Out: %lu\n", s2n_connection_get_wire_bytes_out(conn));
```

Comment:
Should these format strings be `%llu`? Also, this punctuation might fit better with the other stuff printed here:
```suggestion
    printf("Wire bytes in: %llu\n", s2n_connection_get_wire_bytes_in(conn));
    printf("Wire bytes out: %llu\n", s2n_connection_get_wire_bytes_out(conn));
```

Reply:
Followed [this recommendation](https://stackoverflow.com/a/9225648) to use `PRIu64` for printing a `uint64_t` . Included your capitalization changes as well.